ui/app/AppLayouts/Chat/components/MessageContextMenu.qml

```diff
@@ -22,6 +23,13 @@ PopupMenu {
     property var fromAuthor: ""
     property var text: ""
     property var emojiReactionsReactedByUser: []
+    subMenuIcons: [
+        {
+            source: Qt.resolvedUrl("../../../../shared/img/copy-to-clipboard-icon"),
```

Comment:
I think this icon doesn't work considering what you showed in your screenshots

Reply:
I'm probably missing something, but the icon is properly rendered in both screenshots

Reply:
The `subMenuIcons` doesn't show. The sub menu is the second menu that appears after hovering "Copy link"

Reply:
Actually, `subMenuIcons` is used for the menu item that opens the submenu as well:

https://github.com/status-im/status-desktop/blob/master/ui/shared/PopupMenu.qml#L52-L53

That's how I got the `copy-to-clipboard-icon` attached to the nested `PopupMenu`

Reply:
Ah ok, no problem. It's not really important anyway, I just wanted to be sure you had the behaviour you wnated